### DISCUSSION TITLE
Problem with some title IDs, force a specific omdb entry?

### DISCUSSION THREAD

**Discussion by BitHappy** [automatic-ripping-machine/automatic-ripping-machine#852] [Q&A]
I'm having some weird trouble with some of my movies identification somehow. So far I've only tried 5-6 things, and most seem to work fine and get the right title and poster and everything but for some reason any Marvel movies I rip get ID'd as something called "Mechanical Marvels: Clockwork Dreams". Iron Man 3 AND Thor: Dark World both keep getting ID'd as that haha.

I've tried both omdb and tmdb, and it seems to work on other movies fine. Again I've only tried a few but Cars, Transformers, and Anchorman all got ID'd properly so I have no idea what's going on.

I know I can enter a name manually, which is what I've done, but it won't let me edit anything else, so the poster is of that other movie, as is the synopsis and all the other identifying meta data. 

Is there any way to put in a specific id tag or something so that it pulls all the info from the correct place? Or do I have to go in after the fact on my other computer with some meta data editor and do everything manually?

oh I'm using the Docker version by the way, not sure if that makes a difference.

Thanks for the help!

**Comment by 1337-server** [comment]
The automatic recognition doesn't always work 100% sadly. Manually fixing is the last resort.

**Reply by BitHappy** [reply]
Yeahh I definitely understand that. I'm having trouble figuring out how to manually fix everything though is the thing? While its ripping I'm able to put in a custom title for the movie which is great, but everything else stays on what it automatically found, like it stays as the same poster for the wrong movie, as well as the same synopsis and year etc. Is there a way to edit that within the software? Or do I need some other like metadata editor or something to use on the files afterwards?

Its weird because if you click on the current job, it will pull up all the info and it has a spot labeled poster_url_auto, but also a spot labeled poster_url_manual, except as far as I can there's no way to input information there. Its definitely possible I'm missing something obvious though haha.
